### PR TITLE
Readme: Add link into Development Guide

### PR DESCRIPTION
**Pull request by onnimonni**
This makes it super clear where to start contributing to the code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,8 @@ these priorities may not be accepted.
 Please familiarize yourself with the [Contribution
 Guide](https://flynn.io/docs/contributing) before contributing.
 
+You can install flynn development environment and tools by following the [Development Guide](https://flynn.io/docs/development).
```
Comment (onnimonni) [inline README.md]: Oh I did notice that in the first place but I seem to forgot to press `cmd+s` on sublime text.

**Comment by titanous** [comment]
Thanks for submitting this Pull Request! The commit message needs a [DCO sign-off](https://flynn.io/docs/contributing#developer%E2%80%99s-certificate-of-origin).

**Comment by titanous** [comment]
You'll need to squash and force-push so that we can get the whole thing and DCO sign-off all in one commit.

**Comment by onnimonni** [comment]
@titanous Done!

**Comment by titanous** [comment]
Thanks!

**Pull request merged**